sdk/keyvault/keyvault-certificates/karma.conf.js

```diff
@@ -28,7 +28,7 @@ module.exports = function(config) {
       // Promise,String.prototype.startsWith,String.prototype.endsWith,String.prototype.repeat,String.prototype.includes,Array.prototype.includes,Object.keys
       "https://cdn.polyfill.io/v2/polyfill.js?features=Promise,String.prototype.startsWith,String.prototype.endsWith,String.prototype.repeat,String.prototype.includes,Array.prototype.includes,Object.keys|always",
       "dist-test/index.browser.js"
-    ].concat(isPlaybackMode() ? ["recordings/browsers/**/*.json"] : []),
+    ].concat((isPlaybackMode() || isSoftRecordMode()) ? ["recordings/browsers/**/*.json"] : []),
```

Comment:
Just calling it out.. a new method `addRecordings()` is gonna be exposed soon from the recorder instead of this complicated block.

Reply:
Thank you for logging this, it was getting late here 🌙